### PR TITLE
Bugfix/lfx 1803 people tab org

### PR DESCRIPTION
**Pull request by emlimlf**
# Changes proposed ✍️

### What
- Used the activity count included in the organization property in the organization profile overview page
- Added activity count column on the People > Overview table
- Added a new property for the filter.vue component called exclude filter list. This common component is processing the "segmentId" in the URL and parsing it as a filter which causes an error.
- Modified the filter value assignment to check for the missing required properties (ex: order)
- Fix the search string passing on the organizations > people tab

### Why
These changes fixes the bugs stated on this ticket: [https://linear.app/lfx/issue/LFX-1803/issues-in-people-and-organization-profiles](https://linear.app/lfx/issue/LFX-1803/issues-in-people-and-organization-profiles)

### How
copilot:walkthrough

## Checklist ✅
- [ ] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screenshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

## Release Notes

- **New Features**
  - Introduced a search functionality for filtering contributors in the organization details.
  - Added a new "Activities" table header in contributor project details.

- **Improvements**
  - Enhanced readability and layout of various components, including contributor and organization details.
  - Improved formatting for popovers and activity counts for better clarity.

- **Bug Fixes**
  - Streamlined data-fetching processes and error handling in several components.

- **Documentation**
  - Updated prop and method signatures for consistency and clarity across components.

### CODE REVIEW

File: frontend/src/modules/organization/components/details/organization-details-contributors.vue [hunk 1]
```diff
@@ -6,6 +6,7 @@
     :search-config="memberSearchFilter"
     :saved-views-config="memberSavedViews"
     :custom-config="customAttributesFilter"
+    :exclude-filters="excludeFilters"
```
Comment (joanagmaia) [inline frontend/src/modules/organization/components/details/organization-details-contributors.vue:9]: Can you revert all logic added to Filter.vue and FilterItem.vue?
You don't need to add the `exclude-filters` for it to work.

What's happening is that whenever you open an organization profile, a "segmentId" parameter is being added to the url.
Then on "Filter.vue", by default we parse all the query params on this component to parse them as filters. However, this segmentId, along with projectGroup shouldn't be accounted for.
I believe the fix can simply be achieved by updating:
"  const {
    projectGroup, segmentId, menu, ...parsedQuery
  } = query;"
  
  Adding "segmentId" in line 193 of Filters.vue

File: scripts/scaffold.yaml [hunk 2]
```diff
@@ -140,7 +140,7 @@ services:
       - crowd-bridge
 
   kafka:
-    image: bitnami/kafka:latest
```
[the remark this answers is not in the document]
Comment (emlimlf) [reply scripts/scaffold.yaml:143]: I've asked @sausage-todd about this one. He was the one who shared the version of kafka with me. The latest version don't work. What I can probably do is just revert this and modify this as needed only.

File: frontend/src/modules/organization/components/details/organization-details-contributors.vue (hunk 1 above)
Comment (emlimlf) [reply frontend/src/modules/organization/components/details/organization-details-contributors.vue:9]: Got it

File: scripts/scaffold.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (emlimlf) [reply scripts/scaffold.yaml]: I modified this and pushed the changes.

File: frontend/src/modules/organization/components/details/organization-details-contributors.vue (hunk 1 above)
Comment (emlimlf) [reply frontend/src/modules/organization/components/details/organization-details-contributors.vue]: done

**Review by joanagmaia — APPROVED** [review]
LGTM 🚀

**Pull request merged**